python/tk-core/README.md

```diff
@@ -1,4 +1,8 @@
-[![Build Status](https://secure.travis-ci.org/shotgunsoftware/tk-core.png?branch=master)](http://travis-ci.org/shotgunsoftware/tk-core)
```

Comment:
I generally integrate tk-core after code review to reduce the noise. But what you did is fine too!